docs/interact/programmability/lua-api.md

```diff
@@ -678,14 +679,25 @@ In addition, the following external libraries are loaded and accessible to scrip
 * The [_cmsgpack_ library](#cmsgpack-library)
 * The [_bitop_ library](#bitop-library)
 
+### <a name="os-library"></a> _os_ library
+
+* Since version: 8.0.0
+* Available in scripts: yes
+* Available in functions: yes
+
+_os_ provides a set of functions for dealing with date, time, and system commands.
+More details can be found in the [Operating System Facilities](https://www.lua.org/manual/5.1/manual.html#5.8).
+Note that for sandbox security, currently only the following os functions is exposed:
+
+* `os.clock()`
+
 ### <a name="struct-library"></a> _struct_ library
 
 * Since version: 2.6.0
 * Available in scripts: yes
 * Available in functions: yes
 
 _struct_ is a library for packing and unpacking C-like structures in Lua.
-It provides the following functions:
```

Comment:
why did you remove this? consistency reasons?

Reply:
sorry, mistake.